library/patient.inc.php

```diff
@@ -1716,7 +1718,9 @@ function get_patient_balance($pid, $with_insurance = false, $eid = false)
                 "deleted IS NULL AND pid = ? AND encounter = ? AND payer_type = 0",
                 array($pid, $encounter)
             );
-            $copay = !empty($insarr[0]['copay']) ? $insarr[0]['copay'] * 1 : 0;
+            // going to comment this out for now since computing future copays doesn't
+            // equate to cash in hand, which shows in the Billing widget in dashboard 4-23-21
+            // $copay = !empty($insarr[0]['copay']) ? $insarr[0]['copay'] * 1 : 0;
             
```

Comment:
for this see these images to understand how a encounter that didn't even have a fee sheet was incorrectly reflecting cash based on the copay entered in demographics
before:
![Screenshot from 2021-04-23 16-35-29](https://user-images.githubusercontent.com/5618372/115928841-ca1c6a00-a454-11eb-947b-e6f6bba45c15.png)
after:
![Screenshot from 2021-04-23 16-56-31](https://user-images.githubusercontent.com/5618372/115928918-e8826580-a454-11eb-904c-6640a3c36025.png)
demos:
![Screenshot from 2021-04-23 16-57-39](https://user-images.githubusercontent.com/5618372/115929009-09e35180-a455-11eb-9180-cc3f6c75dbde.png)